src/Symfony/Component/HttpClient/composer.json

```diff
@@ -16,6 +16,7 @@
     ],
     "provide": {
         "php-http/client-implementation": "*",
+        "php-http/async-client-implementation": "*",
```

Comment:
actually using alpha order would be nice :)

Reply:
Wasn't sure if I had to do it, but fine. :)